qt-models/filtermodels.h

```diff
@@ -16,16 +16,17 @@ struct dive_trip;
 class DiveTripModel;
 
 struct FilterData {
-	int minVisibility;
-	int maxVisibility;
-	int minRating;
-	int maxRating;
-	double minWaterTemp;
-	double maxWaterTemp;
-	double minAirTemp;
-	double maxAirTemp;
+	bool validFilter = false;
+	int minVisibility = 0;
+	int maxVisibility = 5;
+	int minRating = 0;
+	int maxRating = 5;
+	double minWaterTemp = 0;
+	double maxWaterTemp = 100;
+	double minAirTemp = 0;
+	double maxAirTemp = 100;
```

Comment:
same with these initializations... just do them as you first introduce that code.

Reply:
BTW: initialization in a struct definition... I didn't know that you can do that. Or is this a C++ only thing?